### ISSUE TITLE
rc3 fails to pass tests

### ISSUE THREAD

**Issue by sixtus** [druid-io/druid#3699]
```
Running io.druid.query.lookup.KafkaLookupExtractorFactoryTest
Tests run: 18, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.103 sec <<< FAILURE! - in io.druid.query.lookup.KafkaLookupExtractorFactoryTest
testStartStop(io.druid.query.lookup.KafkaLookupExtractorFactoryTest)  Time elapsed: 0.018 sec  <<< FAILURE!
java.lang.AssertionError:
  Unexpected method call ConsumerConnector.shutdown():
    ConsumerConnector.shutdown(): expected: 2, actual: 3
	at org.easymock.internal.MockInvocationHandler.invoke(MockInvocationHandler.java:44)
	at org.easymock.internal.ObjectMethodsFilter.invoke(ObjectMethodsFilter.java:94)
	at com.sun.proxy.$Proxy34.shutdown(Unknown Source)
	at io.druid.query.lookup.KafkaLookupExtractorFactory.close(KafkaLookupExtractorFactory.java:311)
	at io.druid.query.lookup.KafkaLookupExtractorFactoryTest.testStartStop(KafkaLookupExtractorFactoryTest.java:305)
```

**Comment by dclim** [comment]
This was fixed in #3604 but wasn't backported into 0.9.2.

**Comment by sixtus** [comment]
looks like a simple

```git cherry-pick 8d1d209599c3aa3ff0711d9ed2e8d57b01b5b856```

does the trick
